agents-api/agents_api/activities/task_steps/prompt_step.py

```diff
@@ -78,15 +77,6 @@ async def prompt_step(context: StepContext) -> StepOutcome:
     # Wrap the prompt in a list if it is not already
     prompt = prompt if isinstance(prompt, list) else [{"role": "user", "content": prompt}]
 
```

Comment:
this file's changes should not go into dev just yet. it'll break stuff. instead #1081 should do that later (same for log step activity)

please roll this file back @whiterabbit1983

Reply:
I reverted this